x-pack/plugins/endpoint/server/routes/resolver/utils/pagination.ts

```diff
@@ -15,12 +15,19 @@ export interface PaginationParams {
   eventID?: string;
 }
 
+export interface PaginatedResults {
+  totals: Record<string, number>;
```

Comment:
❔ Is this that `NodePagination` type I saw back up there? Should it be typed with that interface?